filament/backend/src/opengl/platforms/PlatformEGLAndroid.cpp

```diff
@@ -233,24 +236,124 @@ Driver* PlatformEGLAndroid::createDriver(void* sharedContext,
 
 PlatformEGLAndroid::ExternalImageEGLAndroid::~ExternalImageEGLAndroid() = default;
```

Comment:
This will need to call `AHardwareBuffer_release` (to balance the `AHardwareBuffer_acquire` we're adding in create)